src/sensesp/net/web/button_handler.cpp

```
@@ -0,0 +1,69 @@
+        }
+        char name_cstr[url_tail.length() + 1];
+        urldecode2(name_cstr, url_tail.c_str());
+        String name(name_cstr);
+
+        const auto& buttons = UIButton::get_ui_buttons();
+        auto it = buttons.find(name);
+        if (it == buttons.end()) {
+          httpd_resp_send_err(req, HTTPD_404_NOT_FOUND,
+                              "No button found with that name");
+          return ESP_FAIL;
+        }
```

Cppcheck (2.21.0)

[performance] Variable 'ha1_' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)